@here/harp-mapview/lib/MapView.ts

```diff
@@ -141,7 +142,9 @@ export enum MapViewEventNames {
     /** Called when the WebGL context is restored. */
     ContextRestored = "webglcontext-restored",
     /** Called when camera position has been changed. */
-    CameraPositionChanged = "camera-changed"
+    CameraPositionChanged = "camera-changed",
+    /** Called when dispose has been called. */
+    Dispose = "dispose"
```

Comment:
Shouldn't it be dispose**d** since the event is emitted after disposing happened

Reply:
Just checked the code and it is called before the actual disposing happens. So please adjust the comment to make it more clear

Reply:
Will do.

Reply:
Will do.